clients/admin-ui/src/pages/dataset/[id].tsx

```diff
@@ -0,0 +1,39 @@
+import { Box, Breadcrumb, BreadcrumbItem, Heading } from "@fidesui/react";
+import type { NextPage } from "next";
+import NextLink from "next/link";
+import { useRouter } from "next/router";
+
+import Layout from "~/features/common/Layout";
+import DatasetCollectionView from "~/features/dataset/DatasetCollectionView";
+
+const DatasetDetail: NextPage = () => {
+  const router = useRouter();
+  const { id } = router.query;
+
+  if (!id) {
+    return <Layout title="Dataset">Dataset not found</Layout>;
+  }
+
+  const fidesKey = Array.isArray(id) ? id[0] : id;
```

Comment:
Oh interesting, so there are times when the id from the router is an array?

Reply:
yeah apparently! I didn't know that either, but typescript told me 🤷‍♀️ I'm guessing it has something to do with how you can have multiple query parameters, i.e. `?param=1&param=2`, though I haven't looked into it.